nautobot_design_builder/signals.py

```diff
@@ -108,7 +107,7 @@ def load_pre_delete_signals():
 load_pre_delete_signals()
 
 
-@receiver(signal=post_delete, sender=DesignInstance)
-def handle_post_delete_design_instance(sender, instance, **kwargs):  # pylint: disable=unused-argument
-    """Cleaning up the Tag created for a design instance."""
-    Tag.objects.get(name=f"Managed by {instance}").delete()
+# @receiver(signal=post_delete, sender=DesignInstance)
```

Comment:
so, are we dropping the Tag per design deployment?  should we leave it optional?

Reply:
Yes, we are dropping the automatic tagging. I have some ideas on how to implement this more generically, but I don't think that the presence of this tag is a requirement to deliver the initial design lifecycle functionality.